test/rmq/bench/conftest.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+import functools
+
+try:
+    import pika
+except ImportError:
+    pika = None
```

Comment:
what is the point of this if your tests _require_ `pika` to be installed? Should either just raise or you raise yourself saying that `pika` is required for these tests

Reply:
Whoops.  This was left from when I originally thought to not even make `pika` a testing requirement and just do a `skip` asking the user to install it if they wanted to perform those benchmarks but then it became complicated to set up and I dropped that idea.
Removed this try/except.